driver/main.c

```diff
@@ -1027,6 +1030,12 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 #ifdef CAPTURE_PAGE_FAULTS
 		ASSERT(g_tracepoint_registered);
 
+		if (g_fault_tracepoint_disabled) {
+			ret = 0;
```

Comment:
We need to return an error here and propagate it to the user. If not, they could run `sysdig --page-faults` and silently fail.